src/super_gradients/training/utils/utils.py

```diff
@@ -1,20 +1,32 @@
 import math
 import time
 from pathlib import Path
-from typing import Mapping, Optional, Tuple, Union
+from typing import Mapping, Optional, Tuple, Union, List
 from zipfile import ZipFile
 import os
 from jsonschema import validate
+import tarfile
+from PIL import Image, ExifTags
 
 import torch
 import torch.nn as nn
 
+
 # These functions changed from torch 1.2 to torch 1.3
 
 import random
 import numpy as np
 from importlib import import_module
 
+from super_gradients.common.abstractions.abstract_logger import get_logger
+
+logger = get_logger(__name__)
+
+# GET ORIENTATION EXIF TAG
+for orientation in ExifTags.TAGS.keys():
```

Comment:
What is this and what is it doing here ?

Reply:
the value of orientation is required to get the image size. This is how super-gradients used to get its value. I wrapped it in a function with cache to have something equivalent but cleaner:
https://github.com/Deci-AI/super-gradients/blob/a735f79c99ebac0dcaa52b352681fd7d2f03adc1/src/super_gradients/training/utils/utils.py#L404-L409